Microsoft.Toolkit.Uwp/Helpers/StorageFileHelper.cs

```diff
@@ -595,7 +595,7 @@ public static async Task<byte[]> ReadBytesAsync(this StorageFile file)
         /// The <see cref="bool"/>, indicating if the subfolders should also be searched through.
         /// </param>
         /// <returns>
-        /// Returns true, if the file exists.
+        /// true if the file exists; otherwise false.
```

Comment:
Let's try to keep the same format as the other files.
```suggestion
        /// <returns><c>true</c> if the file exists; otherwise, <c>false</c>.</returns>
```

Reply:
done: ab0a3046d8f045416ff3e8815a5f19e82de0be3c
I left the `returns` tags on separate lines if that's how they were before.

Most of the tags in `StorageFileHelper.cs` are on separate lines, which is inconsistent with other files. Might be worth cleaning that up in another PR